<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QPushButton" name="m_openButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="m_playButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="m_stopButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
  </widget>
  <widget class="QSlider" name="m_slider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>419</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="m_tvButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/tv.png</normaloff>:/images/tv.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="m_exitButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
